jdbc-sink/src/test/java/org/springframework/cloud/stream/module/jdbc/ShorthandMapConverterTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.jdbc;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.collection.IsMapContaining.*;
+import static org.junit.Assert.*;
```

Comment:
This is weird. Unless I am mistaken, I am using the sanctioned `spring-framework.xml` IDEA settings, which have static star imports set at 1:
![image](https://cloud.githubusercontent.com/assets/313494/10817637/781f327c-7e3b-11e5-9c69-316e54b6d660.png)

Or is this IDEA playing tricks on me again?